app-config.yaml

```diff
@@ -2,12 +2,17 @@ app:
   title: Backstage Example App
   baseUrl: http://localhost:3000
   googleAnalyticsTrackingId: # UA-000000-0
+  # https:
+  #   certificate:
+  #     cert:
+  #       $file: '\path\to\the\certificate'
+  #     key:
+  #       $file: '\path\to\the\private-key'
```

Comment:
Do you feel strongly that this should be here? I'm feeling it would be alright to keep config as it was. This is a pretty niche use-case

Reply:
You're right, removed 👍